config/config.go

```diff
@@ -0,0 +1,78 @@
+package config
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/appnexus/ankh/context"
+	"github.com/appnexus/ankh/kubectl"
+)
+
+type ConfigMap struct {
+	Data map[string]interface{} `yaml:"data"`
+}
+
+func GetAnkhConfig(ctx *ankh.ExecutionContext, configPath string) (ankh.AnkhConfig, error) {
+	ankhConfig := ankh.AnkhConfig{}
+
+	body := []byte{}
+	var err error
+	if strings.HasPrefix(configPath, "http://") || strings.HasPrefix(configPath, "https://") {
```

Comment:
Let's use `net/url`s `Parse` to pick apart the URL without as much string manipulation.

```
package main

import (
	"fmt"
	"net/url"
)

func main() {
	parsed, _ := url.Parse("kubecontext://foo/bar/baz")
	fmt.Println(parsed.Scheme)
	fmt.Println(parsed.Host)
	fmt.Println(parsed.Path)
}
```

```
kubecontext
foo
/bar/baz

Program exited.
```
https://play.golang.org/p/y3lF6zUUuTm

Reply:
good call